src/components/rent-schedule/tables/rent-schedule-property-table.tsx

```diff
@@ -13,6 +15,7 @@ const useStyles = makeStyles((theme) => ({
 
 const RentSchedulePropertyTable = (property: PropertyModel) => {
     const classes = useStyles();
+    const owner = useAppSelector(selectPersons).filter((person) => person.id === property.owner)[0];
```

Comment:
Dafür gibts einen eigenen Selector:
```suggestion
    const owner = useAppSelector(selectPersonsOwners).filter((person) => person.id === property.owner)[0];
```